src/content/docs/book/part-1-instructions/3-control-flow/3-explore/3-1-calculator.md

```diff
@@ -19,13 +19,13 @@ Average: 10.0
 Add another value: [y/n] y
 Enter value: 30
 Count: 2
-Total: 30.0
+Total: 40.0
 Min: 10.0
 Max: 30.0
 Average: 15.0
```

Comment:
I think average should be 20 here? 
$\frac{\textrm{total}}{\textrm{count}}\equiv\frac{40}{2}=20$

Reply:
Yes that's correct! I missed that